sim/firesim-lib/src/main/scala/passes/EC2F1Artefacts.scala

```diff
@@ -22,15 +22,6 @@ object EC2F1Artefacts extends Transform {
   def outputForm: CircuitForm = LowForm
   override def name = "[Golden Gate] EC2 F1 Artefact Generation"
 
-  // Capture FPGA-toolflow related verilog defines
-  def verilogHeaderAnno = GoldenGateOutputFileAnnotation(
-    """| // Strips out Chisel's default $fatal emission which Vivado chokes on
-       | `define SYNTHESIS
```

Comment:
SYNTHESIS has been removed, and RANDOM has been pushed into the statically set defines in aws-fpga.